parser/src/main.rs

```diff
@@ -129,12 +52,12 @@ fn main() {
             .help("Only types the input"))
         .get_matches();
 
-    let success = if !matches.is_present("test") {
+    let success = {
         let file_name = matches.value_of("input").unwrap();
-        let _parse_only = matches.is_present("parse_only");
-        let _type_only = matches.is_present("type_only");
+        let _parse_only = matches.is_present("parse-only");
+        let _type_only = matches.is_present("type-only");
```

Comment:
You can remove the underscores in front of `_parse_only` and `_type_only` (these two variables are now used).

Reply:
Addressed.